diff --git a/pyqode/python/backend/workers.py b/pyqode/python/backend/workers.py
index 6ff02c2..0171d05 100644
--- a/pyqode/python/backend/workers.py
+++ b/pyqode/python/backend/workers.py
@@ -176,7 +176,6 @@ def run_pep8(request_data):
 
 PYFLAKES_ERROR_MESSAGES = [
     messages.DoctestSyntaxError,
-    messages.ReturnWithArgsInsideGenerator,
     messages.UndefinedExport,
     messages.UndefinedName,
     messages.UndefinedLocal
